File: app/src/main/java/ru/samtakoy/core/data/network/RequestApi.java
package ru.samtakoy.core.data.network;

import io.reactivex.Observable;
import retrofit2.http.GET;
import ru.samtakoy.core.data.network.pojo.RemoteFilesInfo;

public interface RequestApi {

    @GET("info.txt")
    Observable<RemoteFilesInfo> getFilesInfo();

}

File: androidApp/src/main/kotlin/ru/samtakoy/core/data/network/RequestApi.kt
package ru.samtakoy.core.data.network

import io.reactivex.Observable
import retrofit2.http.GET
import ru.samtakoy.core.data.network.pojo.RemoteFilesInfo

interface RequestApi {
    @get:GET("info.txt")
    val filesInfo: Observable<RemoteFilesInfo>
}
